packages/bpk-docs/src/pages/BadgePage/badge-page.scss

```diff
@@ -20,13 +20,24 @@
 
 .bpkdocs-badge-page {
   &__badge-spacing {
+    max-width: 100%;
```

Comment:
This new style doesn't seem to be necessary. Is margin top here only to account for the margin we're setting in the `dark` class?